stat/testuv/norm.go

```diff
@@ -0,0 +1,77 @@
+package testuv
+
+import (
+	"fmt"
+	"math"
+
+	"gonum.org/v1/gonum/stat"
+)
+
+// NormalSkewTest test whether the skew is different from the normal distribution.
+// This function tests the null hypothesis that the skewness of population that the
+// sample was drawn from is the same as that of a corresponding normal distribution.
+// The length of a must be at least 8.
+func NormalSkewTest(a []float64) float64 {
+	// Implementation based on scipy's: https://github.com/scipy/scipy/blob/v1.3.0/scipy/stats/stats.py#L1481-L1549
+	b2 := stat.Skew(a, nil)
+	n := len(a)
+	if n < 8 {
```

Comment:
I would write this a `len(a) < 8` rather than the intermediate variable. `len(x)` is a constant time operation as costly as `n`. Then you can do the `n := float64(len(a))` below and not need an `nf`.